charts/atlantis/templates/statefulset.yaml

```diff
@@ -138,6 +138,12 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if or .Values.gitconfig .Values.gitconfigSecretName }}
```

Comment:
You mentioned

>I believe this should fix the issues in #222.

Could you test out your version of the chart and confirm this works as you intend it to?

Reply:
@kpettijohn

Reply:
I have not had the time to verify the changes at work but hopefully will after the holidays.

Reply:
Please verify before we merge

Reply:
@kpettijohn friendly ping

Reply:
If we cannot test this and get this change (or similar) merged, we'll have to revert #203 @kpettijohn .

If #203 is reverted, @nxf5025 would you be able to take this over to resubmit #203 and prevent #222?

Reply:
@nitrocode That's fine with me

Reply:
Hey, reverting to using `lifecycle.PostStart` for setting .gitconfig could clash with already merged #220